Formula/l/lv.rb

```diff
@@ -75,24 +75,3 @@ def install
      SIDX = index;
  
      if( 'm' != ch ){
---- a/src/guess.c
-+++ b/src/guess.c
-@@ -21,7 +21,7 @@
-  */
-
- #include <stdio.h>
--
-+#include <string.h>
- #include <import.h>
- #include <decode.h>
- #include <big5.h>
---- a/src/guesslocale.c
-+++ b/src/guesslocale.c
-@@ -24,6 +24,7 @@
-
- #include <stdlib.h>
- #include <string.h>
-+#include <ctype.h>
- #include <locale.h>
- #if defined(HAVE_LANGINFO_CODESET)
- #include <langinfo.h>
```

Comment:
These `#include` fixes are included in https://salsa.debian.org/debian/lv/-/blob/debian/4.51-9/debian/patches/fix_implicit_decls.diff?ref_type=tags